model/generate_test.go

```diff
@@ -1373,7 +1373,7 @@ func TestSimulateNewDependencyGraph(t *testing.T) {
 				},
 			},
 		}
-		g.SetNewTasksAndActivationInfo(context.Background(), v, project)
+		g.GetNewTasksAndActivationInfo(context.Background(), v, project)
```

Comment:
nit: since all these tests are using `context.Background()`, I would recommend just initializing the context once at the top of the test:
```go
ctx, cancel := context.WithCancel(context.Background())
defer cancel()

// Test...
```